spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/DataFlowServerConfiguration.java

```diff
@@ -69,7 +68,6 @@
 		DataFlowTaskConfiguration.class
 })
 @EnableConfigurationProperties({ BatchProperties.class, CommonApplicationProperties.class })
-@ComponentScan(basePackages = {"org.springframework.cloud.dataflow.schema.service", "org.springframework.cloud.dataflow.aggregate.task"})
```

Comment:
I believe this was needed when we enabled DataFlowServer configuration as AutoConfiguration which then be used by any other custom DataFlow server implementations. We moved away from supporting customised ones. But, we can always revisit when needed. 

LGTM

Reply:
I believe if someone wants to customize they can add those to their application configuration.